src/components/ProtestForm/ProtestForm.js

```
@@ -78,35 +82,37 @@ function ProtestForm({ initialCoords, submitCallback, defaultValues = {}, afterS
   }, [coordinatesUpdater]);
 
   const onSubmit = async (params) => {
-    if (!params.streetAddress) {
-      alert('אנא הזינו את כתובת ההפגנה');
-      return;
-    } else {
-      try {
-        params.coords = mapCenter;
-        // params.recaptchaToken = recaptchaToken;
-
-        let protest = await submitCallback(params);
-
-        if (editMode) {
-          setSubmitSuccess(true);
-          setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
-          return;
-        }
-
-        if (protest._document) {
-          setSubmitSuccess(true);
-          setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
-          afterSubmitCallback();
-        } else {
-          throw protest;
-        }
-      } catch (err) {
-        console.log('error!!', err);
+    // if (!params.streetAddress) {
+    //   alert('אנא הזינו את כתובת ההפגנה');
+    //   return;
+    // } else {
+    try {
+      params.coords = mapCenter;
+      params.dateTimeList = dateTimeList;
+      // params.recaptchaToken = recaptchaToken;
+
+      let protest = await submitCallback(params);
+
+      if (editMode) {
+        setSubmitSuccess(true);
+        setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
+        afterSubmitCallback();
+        return;
+      }
+
+      if (protest._document) {
         setSubmitSuccess(true);
-        setSubmitMessage('תקלה התרחשה בתהליך השליחה. אנא פנו אלינו וננסה להבין את הבעיה: support@1km.zendesk.com');
+        setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
+        afterSubmitCallback();
+      } else {
+        throw protest;
```

@uriklar why `throw protest`? maybe throw an error?

---

Done